fiboa_cli/datasets/fi.py

```diff
@@ -24,11 +24,17 @@
     "KASVIKOODI_SELITE_FI": "crop_name",
 }
 
+COLUMN_FILTERS = {
+    "PINTA_ALA": lambda col: col > 0.0  # fiboa validator requires area > 0.0
```

Comment:
Would it make more sense to set invalid values to None/Null instead of filtering the data out? Not sure whether the fields are actually invalid or just have invalid areas computed...

Reply:
Yes, I've created an alternative...